Add Read the Docs (#2424)
sindresorhus/awesome: readme.md

- [WebXR](https://github.com/msub2/awesome-webxr#readme) - Enables immersive virtual reality and augmented reality content on the web.
- [OpenStreetMap](https://github.com/osmlab/awesome-openstreetmap#readme) - An open data mapping project utilized by many apps and devices.
- [Computational Biology](https://github.com/inoue0426/awesome-computational-biology#readme) - Computational approaches applied to problems in biology.

> - [Read the Docs](https://github.com/readthedocs-examples/awesome-read-the-docs#readme) - Example documentation projects to inspire and help bootstrap new documentation projects.

Reviewer: The title here is not consistent with the readme title in https://github.com/readthedocs-examples/awesome-read-the-docs#readme It includes the word "Projects"

Author: Ay! Thanks! I had a blind spot there - it's best to call it "Awesome Read the Docs"... being a "project" seems very obvious here, everything in the list is a "project" :)

> - [Read the Docs](https://github.com/readthedocs-examples/awesome-read-the-docs#readme) - Documentation projects built and published on Read the Docs, topics include both science and software.

Author: An alternative that might give more context:

```suggestion
- [Read the Docs](https://github.com/readthedocs-examples/awesome-read-the-docs#readme) - Documentation projects for software, science and academia, built and published on Read the Docs Community and with accompanying source repositories containing their contents and configuration.
```

Author: Any reviewers that would have a helpful opinion here?

Reviewer 2: I think it is a bit long. It could be shorter like you have in your header `Example of Read the Docs for learning and bootstrapping documentation projects.`

Author: You are right, it's too long :+1: 

```suggestion
- [Read the Docs](https://github.com/readthedocs-examples/awesome-read-the-docs#readme) - Example documentation projects for inspiration, learning and bootstrapping new documentation projects on Read the Docs.
```

> - [Read the Docs](https://github.com/readthedocs-examples/awesome-read-the-docs#readme) - Example documentation projects for inspiration, learning and bootstrapping new documentation projects on Read the Docs.

Reviewer: Would be great to shorten the description a tiny bit.

Author: Thanks for the review @sindresorhus - here is a shorter candidate:

```suggestion
- [Read the Docs](https://github.com/readthedocs-examples/awesome-read-the-docs#readme) - Example documentation projects to inspire and help bootstrap new documentation projects.
```

Reviewer 3: Let's add a contributing file, so we can work better, together!
